packages/deployer/package.json

```diff
@@ -10,12 +10,13 @@
     "dist"
   ],
   "scripts": {
-    "build": "yarn clean && yarn compile && yarn webpack && yarn copy:webpack && yarn copy:terraform",
+    "build": "yarn clean && yarn compile && yarn make-cli-executable && yarn webpack && yarn copy:webpack && yarn copy:terraform",
     "clean": "rimraf -rf ./dist *.tgz",
     "cli": "ts-node bin/deployer.ts",
     "compile": "tsc -p tsconfig.build.json",
     "copy:terraform": "copyfiles terraform/**/**/*.tf terraform/**/**/*.tpl  dist/",
     "copy:webpack": "copyfiles .webpack/**/**/*.js dist/",
+    "make-cli-executable": "chmod +x dist/bin/deployer.js",
```

Comment:
Do we want/need this to work on Windows? I'm not sure if building is possible now on Windows though, but this will almost certainly break it (?)

Reply:
Yes, this will most definitely break build on Windows :sweat_smile: Maybe we can use Node's [`fs.chmod`](https://nodejs.org/api/fs.html#fs_fs_chmod_path_mode_callback) instead. I guess they have that covered.

Reply:
Yeah, good point. 

But maybe we could drop the windows support and recommend people to use WSL2?